yt/utilities/amr_kdtree/amr_kdtree.py

```diff
@@ -294,7 +294,7 @@ def get_reduce_owners(self):
                 owners[temp.node_id] = owners[temp.left.node_id]
         return owners
 
-    def reduce_tree_images(self, image, viewpoint):
+    def reduce_tree_images(self, image, viewpoint, use_opacity=True):
```

Comment:
Let's make this keyword-only too
```suggestion
    def reduce_tree_images(self, image, viewpoint, *, use_opacity=True):
```